web/packages/teleterm/src/ui/AccessRequests/AccessRequestsContext.tsx

```diff
@@ -55,13 +56,12 @@ export const AccessRequestsContextProvider: FC<
     'clustersService',
     useCallback(state => state.clusters.get(rootClusterUri), [rootClusterUri])
   );
-  const assumed = clustersService.getAssumedRequests(rootClusterUri);
-
-  const canUse = !!(
-    rootCluster?.features?.advancedAccessWorkflows ||
-    Object.keys(assumed).length
+  const assumed = new Map(
+    Object.entries(clustersService.getAssumedRequests(rootClusterUri))
```

Comment:
I didn't realize that `Record` was used for `assumed` because this data comes from the clustersService. 🙄 I guess down the road we might be able to change `ClustersService` to use a map instead of an object too.

Reply:
In general we should change how we keep the assumed requests in `ClustersService` (now we extend the loggedInUser), but it was too much work, so I didn't touch it.